src/tree/tenants/TenantResourceBranchDataProviderManager.ts

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { BranchDataProvider, ResourceModelBase } from '../../../api/src/index';
+import { ResourceBranchDataProviderManagerBase } from "../ResourceBranchDataProviderManagerBase";
+import { TenantResource } from './tenant';
+
+export class TenantResourceBranchDataProviderManager extends ResourceBranchDataProviderManagerBase<string, BranchDataProvider<TenantResource, ResourceModelBase>> {
+    constructor(
+        defaultProvider: BranchDataProvider<TenantResource, ResourceModelBase>
+    ) {
+        super(defaultProvider, () => { return 'tenant' });
```

Comment:
Is this 'tenant' value used anywhere else? If so it should be a constant.

Reply:
Seems like this is the only place this is used.